Stylesheets/Components/image.css

```diff
@@ -0,0 +1,17 @@
+.sq-img{
+    height:10rem;
+    width:10rem;
```

Comment:
correct me If I am wrong but I think giving fixed width & height will not allow the image to scale properly

Reply:
Hi vanshita,
are you suggesting that I use min/max height/width?